scripts/DllManipulatorScript.cs

```diff
@@ -84,12 +87,29 @@ private void Initialize()
         {
             var initTimer = System.Diagnostics.Stopwatch.StartNew();
 
-            DllManipulator.Options = Options;
-            DllManipulator.Initialize(Thread.CurrentThread.ManagedThreadId, Application.dataPath);
+            DllManipulator.Initialize(Options, Thread.CurrentThread.ManagedThreadId, Application.dataPath);
 
             initTimer.Stop();
             InitializationTime = initTimer.Elapsed;
         }
+
+        /// <summary>
+        /// Will reset the DllManipulator and Initialize it again.
+        /// Note: Unloads all Dlls, may be a dangerous operation if using preloaded
+        /// </summary>
+        public void Reinitialize()
+        {
+            if(_singletonInstance != this)
+                return;
+            
+            if (DllManipulator.Options != null)
```

Comment:
I'm not certain, but it appears to me that we could reduce the number of different places when options are stored and move them from this class to the `DllManipulator` itself? TBH I don't really like how it's scattered.

Reply:
Yes its a bit scattered but I think this makes sense:
1. DllManipulatorScript has an Options copy which are seen in the inspector.
1. DllManipulator has Options which are currently used.
1. DllManipulatorEditor has Options solely for detecting changes in the inspector.
Each .cs only has to consider one Options (except the editor). Having the previousInspectorOptions from the editor in DM means DM has a copy which it never uses.

Reply:
ya, sure